cmd/influxd/restore_test.go

```diff
@@ -84,6 +88,7 @@ func TestRestoreCommand(t *testing.T) {
 	f.Close()
 
 	// Stop server.
+	l.Close()
 	s.Close()
 	b.Close()
```

Comment:
Can you defer the closing right after you validate that `b`, `s`, & `l` are non-nil?